.github/actions/gatsby-site-showcase-validator/README.md

```diff
@@ -1,5 +1,5 @@
 # Gatsby Site Showcase Validator
 
-A simple node script that visits and checks all of the sites in the [Site Showcase](https://www.gatsbyjs.org/showcase/) for Gatsby.
+A simple node script that visits and checks all of the sites in the [Site Showcase](https://www.gatsbyjs.or/showcase/) for Gatsby.
```

Comment:
Small typo snuck in here. ✌️

Reply:
you review faster then i can send a fix ;)